custom.props

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup Label="Version">
-    <VersionMajor>2</VersionMajor>
-    <VersionMinor>7</VersionMinor>
+    <VersionMajor>3</VersionMajor>
```

Comment:
>VersionMajor>3</VersionMaj [](http://example.com/codeflow?start=5&length=26)

so far we have been releasing 2.8.0-beta versions for android and iOS, are we sure we should be moving to version 3.0.0?

Reply:
per standup, we're cool with this on UWP